setup.py

```diff
@@ -6,13 +6,14 @@
 
 setup(
     name="Open edX",
-    version="0.4",
+    version="0.5",
```

Comment:
@andy-armstrong @jimabramson @wedaly I have bumped up the version of `Open-edX` due to addition of a new starting point/package `VerificationPartitionScheme`.